lib/Server.js

```diff
@@ -389,6 +389,23 @@ function Server(compiler, options) {
 			const pems = selfsigned.generate(attrs, {
 				algorithm: "sha256",
 				days: 30,
+				extensions: [{
+					name: "subjectAltName",
+					altNames: [
+						{
+							type: 2, // DNS
+							value: "localhost"
+						},
+						{
+							type: 6, // URI
```

Comment:
Reason I ask: I'm not sure which browser specifically wants a URI entry type here (and is not satisfied by the DNS `localhost` entry above)